### DISCUSSION TITLE
Introspect JSON fields  on relationnal database when prisma pull

### DISCUSSION THREAD

**Discussion by gpoumarat** [prisma/prisma#25665] [Q&A]
### Question

I'd like to know if the capability to instrospect databases with JSON field to create the sub schemas is something you'd like to work on or not.
I got actually a database with json data and i'd like to retrieve the schema inside the json data and actually it doesn't work because --composite-type-depth is only for document databases. 

### Information about Prisma Schema, Client Queries and Environment (optional)

```prisma
// Add your schema.prisma
```

```ts
// Add any relevant Prisma Client queries here
```

- OS: 
- Database: 
- Node.js version: 

- Run `prisma -v` to see your Prisma version and paste it

**Comment by ludralph** [comment]
Hi @gpoumarat 

Please open a [feature request](https://github.com/prisma/prisma/issues/new?assignees=&labels=&template=feature_request.md&title=) and document your usecase to help our engineering team evaluate its feasibility. I also suspect it may be related to this other [issue](https://github.com/prisma/prisma/issues/3219).

**Reply by gpoumarat** [reply]
Yes totally related to the linked  issue but in the issue , it's the declarative approach when you declare your model.
I'd like to allow the introspect function in the prisma db pull to analyse json fields and generate the schema inside the json if it's possible. I'll open a feature request.

**Reply by gpoumarat** [reply]
Here the feature request : https://github.com/prisma/prisma/issues/25666

**Reply by ludralph** [reply]
Thanks for creating the feature request. I'll close this discussion.
